kit/controls/src/index.ts

```diff
@@ -26,6 +26,7 @@ export * from './UIShowMoreButton';
 export * from './UISwitcher';
 export * from './UIWideBoxButton';
 export * from './UIPressableArea';
+export * from './Pressable';
```

Comment:
`Pressable` is an internal module. It is not worth exporting, because it has its own peculiarities of use. In order to make the area tappable, you can use a ready-made `UIPressableArea` component.

Reply:
@SAnatoliiS it's just we need to reimplement the button completely in Surf since @eteslov provided a new design)) That's why @vetkamak had to use "private" methods ;)

Reply:
Are you really sure that you need to create buttons in `Surf` and not in `UIKit`?

Reply:
What do you guys think about to move it to UIKit? Shoudn't be hard to do. What cons do you see?

Reply:
I suppose there is no need in moving it to the UIKit - if I correctly understand we will update colors and current buttons soon and added Button will be removed after that. So it is a kind of temporary solution in order to fit the design, after updates we can turn Pressable back to an internal one